interactive-calculator/src/App.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import './App.css';
+
+class App extends Component {
+  state = {
+    total: "",
+    service:'',
+    partySize: '',
+    totalTip: '',
+    tipPP: ''
+  }
+
+  handleTotal = e => {
+    this.setState({
+      total: e.target.value
+    })
+  }
+  handleService = e => {
+    this.setState({
+      service: e.target.value
+    })
+  }
+  handleParty = e => {
+    this.setState({
+      partySize: e.target.value
+    })
+  }
+  genTip = e => {
+    e.preventDefault()
+    let tip = parseFloat(this.state.total * this.state.service).toFixed(2)
+    let totalTipPP = parseFloat(tip / this.state.partySize).toFixed(2)
+    this.setState({
+      totalTip: tip,
+      tipPP: totalTipPP
+    })
+  }
+  render() {
+    return (
+      <div className="App">
+        <header className="App-header">
+          <h1>Just the Tip Calculator</h1>
+        </header>
+        <main className='App-main'>
+          <form className='tip-form' onSubmit={this.genTip}>
+            <label><span className='label-header'>Bill Total: </span>
+              <input type='text' value={this.state.total} onChange={this.handleTotal}/>
+            </label>
+            <label><span className='label-header'>Service: </span>
+              <select onChange={this.handleService}>
+                <option value=''>Select Tip</option>
+                {[.05, .10, .15, .20, .25, .30, .35, .40, .45, .50].map((service, i) => <option key={i} value={service} >{service}</option>)}
```

Comment:
You should just make a variable for this shouldn't even take you that long so that when you need to change the value you just change the variable and don't have to come looking down here for the data.